numba/cuda/tests/cudapy/test_intrinsics.py

```diff
@@ -183,6 +187,18 @@ def foo(out):
 
         self.assertTrue(np.all(arr))
 
+    def test_popc_u4(self):
+        compiled = cuda.jit("void(int32[:], uint32)")(simple_popc)
+        ary = np.zeros(1, dtype=np.int32)
+        compiled(ary, 0xF0)
+        self.assertTrue(ary[0] == 4)
```

Comment:
can you use `self.assertEqual(ary[0], 4)` instead?

Reply:
also, a bit more testing to show that popc doesn't always return 4.

Reply:
@sklam done!